Add `Spanish Showdown 2021` wiki article (#7846)
ppy/osu-wiki: wiki/Tournaments/SS/2021/en.md

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2021-02-21/2021-03-19 |
| Qualifiers | 2021-03-26/2021-03-28 |
| Round of 48 | 2021-04-08/2021-04-11 |
| Round of 32 | 2021-04-11/2021-04-12 |
| Round of 16 | 2021-04-17/2021-04-20 |
| Quarterfinals | 2021-04-23/2021-04-25 |
| Semifinals | 2021-05-01/2021-05-03 |
| Finals | 2021-05-08/2021-05-09 |
| Grand Finals | 2021-05-15/2021-05-16 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Exclusive banner, exclusive SS Discord server role, €52.5 |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | Exclusive banner, €35 |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | Exclusive banner, €17.5 |
| *4th place* | Exclusive banner |
| *5th place* | Exclusive banner |
| *6th place* | Exclusive banner |
| *7th place* | Exclusive banner |
| *8th place* | Exclusive banner |

> Total prize pool was €105 being splitted evenly by the decision from organiser, for exact formula please see the [SS 2021 discussion thread](https://osu.ppy.sh/community/forums/topics/1254577).

Reviewer: ```suggestion
The total prize pool was €105 split evenly by the decision of the organiser. For the exact formula see the [SS 2021 discussion thread](https://osu.ppy.sh/community/forums/topics/1254577).
```

| **Kalal331** ::{ flag=ES }:: | **6** | 1 | ::{ flag=ES }:: G U R A S I M P | [#1](https://osu.ppy.sh/community/matches/82308854) |

Sunday, 2 May 2021:

| Player 1 |  |  | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| amborgesa ::{ flag=ES }:: | 4 | **6** | ::{ flag=ES }:: **NHarmonia** | [#1](https://osu.ppy.sh/community/matches/82317569) |
| Kalal331 ::{ flag=ES }:: | 5 | **6** | ::{ flag=ES }:: **kazamabc** | [#1](https://osu.ppy.sh/community/matches/82367432) |
| **Estopa** ::{ flag=ES }:: | **6** | 3 | ::{ flag=ES }:: Rekens | [#1](https://osu.ppy.sh/community/matches/82371464) |
| A N T O N I O ::{ flag=ES }:: | 5 | **6** | ::{ flag=ES }:: **waaiiru** | [#1](https://osu.ppy.sh/community/matches/82376004) |
| NHarmonia ::{ flag=ES }:: | 5 | **6** | ::{ flag=ES }:: **-Hideki-** | [#1](https://osu.ppy.sh/community/matches/82447571) |

### Quarterfinals

Friday, 23 April 2021:

| Player 1 |  |  | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **Aricin Fanboy** ::{ flag=ES }:: | **0** | -1 | ::{ flag=ES }:: Akalude | *win by default* |

> | Bairechuwu ::{ flag=ES }:: | **6** | 0 | ::{ flag=ES }:: Osudroid96 | [#1](https://osu.ppy.sh/community/matches/81584199) |

Reviewer: ```suggestion
| **Bairechuwu** ::{ flag=ES }:: | **6** | 0 | ::{ flag=ES }:: Osudroid96 | [#1](https://osu.ppy.sh/community/matches/81584199) |
```

| **-Vappy** ::{ flag=ES }:: | **5** | 1 | ::{ flag=ES }:: S\_Neke | [#1](https://osu.ppy.sh/community/matches/80483949) |
| Radiownd ::{ flag=ES }:: | 1 | **5** | ::{ flag=ES }:: **B of yassin** | [#1](https://osu.ppy.sh/community/matches/80488845) |
| **dariito0xx** ::{ flag=ES }:: | **5** | 0 | ::{ flag=ES }:: xCepac | [#1](https://osu.ppy.sh/community/matches/80494402) |
| **AbejaIvan** ::{ flag=ES }:: | **5** | 2 | ::{ flag=ES }:: \[-Chitoge-\] | [#1](https://osu.ppy.sh/community/matches/80495438) |
| **Quantum Fool** ::{ flag=ES }:: | **5** | 3 | ::{ flag=ES }:: Kurumi- | [#1](https://osu.ppy.sh/community/matches/80512506) |
| -Aleph ::{ flag=ES }:: | -1 | **0** | ::{ flag=ES }:: **krYxUs5** | *win by default* |

Sunday, 11 April 2021:

| Player 1 |  |  | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **ImMyyrh** ::{ flag=ES }:: | **5** | 1 | ::{ flag=ES }:: Jerkie | [#1](https://osu.ppy.sh/community/matches/80512532) |
| Elektro ::{ flag=ES }:: | 4 | **5** | ::{ flag=ES }:: **Osudroid96** | [#1](https://osu.ppy.sh/community/matches/80567347) |

### Qualifiers

> Detailed statistics for this round can be found [here](https://docs.google.com/spreadsheets/d/1Wmgzv8OY1_cVCFSAlTFSK4RRYaeP4pqsvyvM6kzsoEw/edit?usp=sharing) alongside the final standings on which players made it through to the next stage.

Reviewer: ```suggestion
Detailed statistics for this round can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1Wmgzv8OY1_cVCFSAlTFSK4RRYaeP4pqsvyvM6kzsoEw/edit?usp=sharing) alongside the final standings on which players made it through to the next stage.
```
